src/ProjectTemplates/test/BlazorServerTemplateTest.cs

```diff
@@ -85,6 +85,7 @@ public async Task BlazorServerTemplateWorks_NoAuth()
         [InlineData(true)]
         [InlineData(false)]
         [SkipOnHelix("ef restore no worky")]
```

Comment:
@HaoK is this still relevant?

Reply:
The skip is still needed because selenium isn't working yet, so the reason is wrong, but the test still doesn't work yet